src/App.js

```diff
@@ -1,25 +1,50 @@
-import logo from './logo.svg';
+import { useState } from 'react'
 import './App.css';
+import foodsJson from "./foods.json";
+import Foodbox from './Components/Foodbox';
+import AddFoodForm from './Components/AddFoodForm';
+import Search from './Components/Search';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+function App () {
+  let [foods, setFoods] = useState(foodsJson);
+  let [foodFilterData, setFoodFilterData] = useState(foods); 
+
+  let addNewFood = (newFood) => {
+    let updatedFoods = [...foodsJson, newFood];
+    setFoods(updatedFoods);
+  }
+
+  let handleSelect = (filteredFood) => {
+    setFoodFilterData(filteredFood);
+  }
 
-export default App;
+  let filterFoodList = (str) => {
+    let filteredFood;
+    if (str === "") {
+      filteredFood = foods;
+    } else {
+      filteredFood = foods.filter((food) => {
+        return food.name.toLowerCase().includes(str.toLowerCase());
+      });
+    }
+    setFoodFilterData(filteredFood);
+  };
+
+  return(
+      <div className="App">
+        <Search searchFood={filterFoodList}/>
+        {foods.map((food) => {
```

Comment:
Now that you have the search bar done you need to map `foodFilterData` instead of `foods`